docs/content/guides/getting-started/setting-options.md

```diff
@@ -663,13 +749,27 @@ hot.getCellMeta(0, 0).readOnly;
 // returns `true`
 hot.getCellMeta(1, 1).readOnly;
 ```
+:::
+
+::: only-for react
+```jsx
+// for cell (0, 0), the `readOnly` option is the default (`false`)
```

Comment:
I think this would implicate the existence of a preconfigured HOT instance:

```suggestion
// Consider the HotTable component with the `cell` option declared:
<HotTable cell={[
  {row: 1, col: 1, readOnly: true }
]}/>

// ...

// for cell (0, 0), the `readOnly` option is the default (`false`)
```

Reply:
Updated according to the snippet